### PR TITLE
10A pre-selects value based on case data

### PR DESCRIPTION
**Pull request by plusjeff**
For issue #296 

From `fakes.rb`, `joe-snuffy` should select `No` and `hearing-requested` should select `Yes`.

### CODE REVIEW

File: app/models/fakes.rb [hunk 1]
```diff
@@ -80,7 +81,11 @@ def bfms=(value)
       end
 
       def hearing_requested?
-        true
+        if bfha && ['1', '2', '6'].include?(bfha)
+          true
+        else
+          false
+        end
```
Comment (alex) [inline app/models/fakes.rb]: You can simplify this to just be:
```ruby
def hearing_requested?
  bfha && ['1', '2', '6'].include?(bfha)
end
```

File: app/views/web/questions.html.erb [hunk 2]
```diff
@@ -187,6 +187,7 @@
               option2_label: 'No',
               option2_value: 'no',
               required: true,
+              selected: @kase.hearing_requested? ? 'yes' : 'no'
```
Comment (alex) [inline app/views/web/questions.html.erb]: Is there a reason not to just pass this as a bool?

**Comment by paultag** [comment]
Can someone also see if you can dupe my bug of 10A 'no' not rendering on the form?

**Comment by paultag** [comment]
Passing this PR to Alex, he gave great feedback on the commit

File: app/models/fakes.rb [hunk 3]
```diff
@@ -306,6 +309,30 @@ def upload_form8(first_name, middle_initial, last_name, file_name)
...
+        efolder_soc: Date.parse('2015-09-01'),
+        efolder_appellant_id: '22222222',
+        appeal_type: 'Post Remand',
+        vso_full: 'Disabled American Veterans',
+        regional_office_full: 'Philadelphia, PA',
+        folder: Folder.new('VBMS'),
+        correspondent: Correspondent.new(
+          'Joe Snuffy', 'Self', 'Joe Snuffy', 'Joe', '', 'Snuffy',
+        ),
+        bfha: '1',
```
Comment (paultag) [inline app/models/fakes.rb]: Just example data; but should we use a const here too?

**Pull request merged**